Assets/HoloToolkit/Utilities/Tests/TextToSpeechManager.unity

```diff
@@ -388,54 +388,6 @@ Light:
   m_BounceIntensity: 1
   m_ShadowRadius: 0
   m_ShadowAngle: 0
---- !u!1 &478672352
```

Comment:
Hmmm.... I should have diffed this before committing. Te scene works fine, but I have no idea what all these deleted lines are about. Is this a problem?